Shoko.Server/Commands/Import/CommandRequest_DownloadAniDBImages.cs

```diff
@@ -28,204 +25,180 @@ namespace Shoko.Server.Commands.Import;
-                        if (!ServerSettings.Instance.AniDb.DownloadCharacters)
-                        {
-                            continue;
-                        }
-
-                        var chrs = (from xref1 in RepoFactory.AniDB_Anime_Character.GetByAnimeID(AnimeID)
-                                select RepoFactory.AniDB_Character.GetByCharID(xref1.CharID))
-                            .Where(a => !string.IsNullOrEmpty(a?.PicName))
-                            .DistinctBy(a => a.CharID)
-                            .ToList();
-                        if (chrs == null || chrs.Count == 0)
-                        {
-                            Logger.LogWarning(
-                                "AniDB Character image failed to download: Can\'t find Character for anime: {AnimeID}",
-                                AnimeID
-                            );
-                            return;
-                        }
-
-                        foreach (var chr in chrs)
-                        {
-                            downloadUrls.Add(string.Format(_handler.ImageServerUrl, chr.PicName));
-                            fileNames.Add(chr.GetPosterPath());
-                        }
-
-                        ShokoService.CmdProcessorImages.QueueState = PrettyDescriptionCharacters;
-                        break;
-
-                    case ImageEntityType.AniDB_Creator:
-                        if (!ServerSettings.Instance.AniDb.DownloadCreators)
-                        {
-                            continue;
-                        }
-
-                        var creators = (from xref1 in RepoFactory.AniDB_Anime_Character.GetByAnimeID(AnimeID)
-                                from xref2 in RepoFactory.AniDB_Character_Seiyuu.GetByCharID(xref1.CharID)
-                                select RepoFactory.AniDB_Seiyuu.GetBySeiyuuID(xref2.SeiyuuID))
-                            .Where(a => !string.IsNullOrEmpty(a?.PicName))
-                            .DistinctBy(a => a.SeiyuuID)
-                            .ToList();
-                        if (creators == null || creators.Count == 0)
-                        {
-                            Logger.LogWarning(
-                                "AniDB Seiyuu image failed to download: Can\'t find Seiyuus for anime: {AnimeID}",
-                                AnimeID
-                            );
-                            return;
-                        }
-
-                        foreach (var creator in creators)
-                        {
-                            downloadUrls.Add(string.Format(_handler.ImageServerUrl, creator.PicName));
-                            fileNames.Add(creator.GetPosterPath());
-                        }
-
-                        ShokoService.CmdProcessorImages.QueueState = PrettyDescriptionCreators;
-                        break;
+                    Logger.LogWarning(FailedToDownloadNoID, "characters for anime", AnimeID);
```

Comment:
No. Don't move new things to resources. We are moving away from them. Also...especially don't mix and match hardcoded language strings and resources...
Please fix all occurrences

Reply:
I'm not using resources here though. This is a `const string` message template to prevent the same line being repeated over and over in the code... but now is the usage of the `const string` repeated over and over instead. The upside is that it is now one line change to update all the messages.